### PR TITLE
ODC Known Issues and Bug Fixes

### PR DESCRIPTION
**Pull request by Preeticp**
This issue adds Known Issues and Bug Fixes to the 4.4 version.
It has received acks from the SMEs

### CODE REVIEW

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -344,6 +344,13 @@ The following related APIs will be removed in a future release:
 [id="ocp-4-4-bug-fixes"]
 == Bug fixes
 
+*Web Console*
+
+* Earlier, when two consecutive rollouts failed, the *Topology* view showed failed pods instead of displaying the last active revision. With this bug fix, when a rollout fails the last active revision is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-4-release-notes.adoc:349]: fails,

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -344,6 +344,13 @@ The following related APIs will be removed in a future release:
...
+
+* Previously, when a user with limited cluster-wide permissions created an application using the *Container Image* option in the *Add* page, and chose the *Image name from the internal registry* option, no imagestreams were detected in the project, though an imagestream existed. Now, the imagestream fetching logic has been moved from the cluster level to the namespace level enabling an user with permission to a namespace to see imagestreams in that namespace.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1784264[*BZ#1784264*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-4-release-notes.adoc:351]: s/using/by using

s/and,/and

The first sentence is too long.

s/level enabling/level, which enables

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -760,3 +767,9 @@ URIs use the `git://` or `ssh://` scheme.
 the mirror registry only links to the builder service account. The pull secret
 must also link to the build config object.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1810904[*BZ#1810904*])
+
+* When a user selects a node or a group of nodes in the *Topology* view on the Firefox browser, the backgrounds of all the associated labels and nodes become transparent. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822337[*BZ#1822337*])
+
+* In  the *Topology view*, when a user selects a node or workload and then clicks the *View monitoring dashboard* link in the *Monitoring* tab on the side panel, the user sees the monitoring dashboard for that specific workload. This filtered workload dashboard view is not clearly named, causing confusion with the generic dashboard that displays metrics for all the workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822331[*BZ#1822331*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-4-release-notes.adoc:773]: There's an extra space after "In"

s/named, causing/named, which causes

File: release_notes/ocp-4-4-release-notes.adoc [hunk 4]
```diff
@@ -760,3 +767,9 @@ URIs use the `git://` or `ssh://` scheme.
...
+
+* In the *Developer* perspective, when invalid characters such as a period (.) are entered in the serverless traffic distribution tag, it stops working, but displays no error messages to prevent invalid characters from being entered in the tag. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822344[*BZ#1822344*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-4-release-notes.adoc:775]: "it stops working," - please clarify what "it" is.  I can't tell if you mean the developer perspective of the distribution tag.

No comma after working.

This sentence is too long.

File: release_notes/ocp-4-4-release-notes.adoc (hunk 2 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: @kalexand-rh I have made the changes and have tried to shorten the sentence, please let me know if this is alright.

File: release_notes/ocp-4-4-release-notes.adoc (hunk 3 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: Fixed

File: release_notes/ocp-4-4-release-notes.adoc (hunk 4 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: Fixed the changes and have shortened the sentence, let me know if this is alright. Thank you!

File: release_notes/ocp-4-4-release-notes.adoc [hunk 5]
```diff
@@ -371,6 +371,13 @@ Package Manager CLI (`opm`), see the link:https://docs.okd.io/latest/operators/u
 [id="ocp-4-4-bug-fixes"]
 == Bug fixes
 
+*Web Console*
+
+* Earlier, when two consecutive rollouts failed, the *Topology* view showed failed pods instead of displaying the last active revision. With this bug fix, when a rollout fails, the last active revision is displayed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1760828[*BZ#1760828*])
```
Comment (bobfuru) [inline release_notes/ocp-4-4-release-notes.adoc:376]: "Earlier" here sounds awkward. Suggest "Previously" or "Before" instead.

File: release_notes/ocp-4-4-release-notes.adoc (hunk 1 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: Fixed

File: release_notes/ocp-4-4-release-notes.adoc [hunk 6]
```diff
@@ -371,6 +371,13 @@ Package Manager CLI (`opm`), see the link:https://docs.okd.io/latest/operators/u
...
+
+* Previously, when a user with limited cluster-wide permissions created an application in the *Add* page, by using the *Container Image* and *Image name from the internal registry* options, no imagestreams were detected in the project, though they existed. Now, the imagestream fetching logic has been moved from the cluster level to the namespace level, which enables a user with permission to a namespace to see imagestreams in that namespace.(link:https://bugzilla.redhat.com/show_bug.cgi?id=1784264[*BZ#1784264*])
```
Comment (bobfuru) [inline release_notes/ocp-4-4-release-notes.adoc:378]: The first sentence is still too long. Suggest breaking into 2-3 sentences, something like:
Previously, existing imagestreams were not detected when creating an application in the "Add" page. This occurred when users with limited cluster-wide permissions used the *Container Image* and *Image name from the internal registry* options.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 7]
```diff
@@ -809,3 +816,9 @@ the client to fail or incorrectly patch the RHV machine status.
 * {product-title} cluster autoscaling on RHV fails due to cluster resource
 computation errors.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822118[*BZ#1822118*])
+
+* When a user selects a node or a group of nodes in the *Topology* view on the Firefox browser, the backgrounds of all the associated labels and nodes become transparent. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822337[*BZ#1822337*])
```
Comment (bobfuru) [inline release_notes/ocp-4-4-release-notes.adoc:820]: s/When a user/When using the Firefox browser to select a node/
s/view on the Firefox browser,/view,/

File: release_notes/ocp-4-4-release-notes.adoc [hunk 8]
```diff
@@ -809,3 +816,9 @@ the client to fail or incorrectly patch the RHV machine status.
...
+
+* In the *Topology view*, when a user selects a node or workload and then clicks the *View monitoring dashboard* link in the *Monitoring* tab on the side panel, the user sees the monitoring dashboard for that specific workload. This filtered workload dashboard view is not clearly named, which causes confusion with the generic dashboard that displays metrics for all the workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822331[*BZ#1822331*])
+
+* In the *Developer* perspective, when invalid characters such as a period (.) are entered in the serverless traffic distribution tag, the traffic distribution feature stops working but displays no error messages to prevent the user from entering invalid characters in the tag. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822344[*BZ#1822344*])
```
Comment (bobfuru) [inline release_notes/ocp-4-4-release-notes.adoc:824]: s/In the/From the/

**Review by bobfuru — COMMENTED** [review]
A few more nits before merging, if you don't mind fixing.

File: release_notes/ocp-4-4-release-notes.adoc (hunk 5 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: Fixed

File: release_notes/ocp-4-4-release-notes.adoc (hunk 7 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: Fixed

File: release_notes/ocp-4-4-release-notes.adoc (hunk 8 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: Reworded.

File: release_notes/ocp-4-4-release-notes.adoc (hunk 6 above)
Comment (Preeticp) [reply release_notes/ocp-4-4-release-notes.adoc]: Thank you @bobfuru. I have tried to shorten all the big sentences in the PR. PTAL and let me know if anything else needs to be modified.

File: release_notes/ocp-4-4-release-notes.adoc [hunk 9]
```diff
@@ -805,3 +812,9 @@ the client to fail or incorrectly patch the RHV machine status.
 * {product-title} cluster autoscaling on RHV fails due to cluster resource
 computation errors.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822118[*BZ#1822118*])
+
+* When using the Firefox browser to select a node or a group of nodes in the *Topology* view, the backgrounds of all the associated labels and nodes become transparent. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822337[*BZ#1822337*])
+
+* In  the *Topology view*, when a user selects a node or workload, and then on the side panel, clicks *Monitoring* -> *View monitoring dashboard*, the user sees the monitoring dashboard for that specific workload. This filtered workload dashboard view is not clearly named, which causes confusion with the generic dashboard that displays metrics for all the workloads. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1822331[*BZ#1822331*])
```
Comment (bobfuru) [inline release_notes/ocp-4-4-release-notes.adoc:818]: Remove extra space ("In the")

File: release_notes/ocp-4-4-release-notes.adoc (hunk 6 above)
Comment (bobfuru) [reply release_notes/ocp-4-4-release-notes.adoc]: > Thank you @bobfuru. I have tried to shorten all the big sentences in the PR. PTAL and let me know if anything else needs to be modified.

Thanks. The revisions look great. Two more small nits, if you don't mind changing?

**Comment by bobfuru** [comment]
LGTM

**Pull request merged**